### PR TITLE
chore(core): disable code sniffer checks

### PR DESCRIPTION
**Pull request by iionly**

### CODE REVIEW

**Comment by iionly** [comment]
Commit as suggested in https://github.com/Elgg/Elgg/pull/10816.

File: composer.json [hunk 1]
```diff
@@ -41,8 +40,6 @@
         }
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.1",
```
Comment (mrclay) [inline composer.json]: Keep this line

File: composer.json [hunk 2]
```diff
@@ -41,8 +40,6 @@
...
-        "elgg/sniffs": "dev-master",
         "squizlabs/php_codesniffer": "~1.5",
```
Comment (mrclay) [inline composer.json]: Remove this line

**Review by mrclay — CHANGES_REQUESTED** [review]

**Comment by iionly** [comment]
Is it correct now?

**Comment by hypeJunction** [comment]
Merge this into master properly right away pleased

**Comment by mrclay** [comment]
Meaning we leave the 2.3 checks in, right?

**Comment by hypeJunction** [comment]
Meaning keep checks intact in master

**Pull request merged**